### ISSUE TITLE
Validation in the authorization form of the username field

### ISSUE THREAD

**Issue by inrosh** [divizia/db-constructor#9]
The validation in the Username field is symbolic in the authorization form when a registered user enters the correct Password, Username  but a Username with end spaces
![authorisation 1](https://user-images.githubusercontent.com/102860611/200803193-5b919120-0f0d-4899-b9f2-e739f55bd15f.png)
![authorisation 2](https://user-images.githubusercontent.com/102860611/200803197-d6f05926-62f9-4df4-b189-e8f6ef99049e.png)

**Comment by lipiridi** [comment]
Yep, we should trim this field (in js) before the request will be sent

**Comment by inrosh** [comment]
I misdescribed it - the user is registered as test1 without spaces, but during another login, I added spaces in the field, and the authorization system missed it.

**Comment by lipiridi** [comment]
I'm already afraid when you will start test the rest api😁

**Comment by inrosh** [comment]
but then we will make a class application👌
